packages/astro/src/runtime/server/astro-island.ts

```diff
@@ -2,6 +2,16 @@
 // Do not import this file directly, instead import the prebuilt one instead.
 // pnpm --filter astro run prebuild
 
+type directiveAstroKeys = string;
```

Comment:
My thinking here was that we might support custom directives in the future and then this might actually be any key. But I didn't really think this all the way through so this might be too confusing as is, will change it.